CMakeLists.txt

```diff
@@ -149,6 +149,39 @@ ELSE (ENABLE_PIE)
     SET(CMAKE_POSITION_INDEPENDENT_CODE OFF)
 ENDIF (ENABLE_PIE)
 
+INCLUDE(CheckIncludeFile)
+INCLUDE(CheckIncludeFileCXX)
+INCLUDE(CheckTypeSize)
+INCLUDE(CMakePushCheckState)
+SET(SIZEOF_VOID_P CMAKE_SIZEOF_VOID_P)
+
+SET(CMAKE_FIND_FRAMEWORK "LAST")
+
+#Find Math
+INCLUDE(CheckSymbolExists)
+IF(NOT POW_FUNCTION_EXISTS AND NOT NEED_LINKING_AGAINST_LIBM)
```

Comment:
I was unable to find a lot of information about this. Is there really a chance modern systems don't have POW? (whatever that is)
I'm asking because a lot of stuff in our cmake was really legacy from 2001.

Reply:
I doubt it. Created #1043 to follow up on that.

Reply:
We need `pow`. This block of code tells us where to find it (i.e., what library). I don't think this code should be removed.